terraform/deployments/govuk-publishing-platform/app_static.tf

```diff
@@ -1,51 +1,98 @@
-  vpc_id                           = local.vpc_id
   cluster_id                       = aws_ecs_cluster.cluster.id
-  source                           = "../../modules/app"
+  vpc_id                           = local.vpc_id
+  subnets                          = local.private_subnets
   desired_count                    = var.static_desired_count
   extra_security_groups            = [local.govuk_management_access_security_group, aws_security_group.mesh_ecs_service.id]
+  environment_variables = merge(
+    local.static_defaults.environment_variables,
+    {
+      ASSET_HOST          = "https://static-ecs.${local.defaults.environment_variables.GOVUK_APP_DOMAIN_EXTERNAL}",
+      GOVUK_STATSD_PREFIX = "govuk-ecs.app.static"
+    },
+  )
+  secrets_from_arns = merge(
+    local.static_defaults.secrets_from_arns,
+    {
+      PUBLISHING_API_BEARER = data.aws_secretsmanager_secret.static_publishing_api_bearer_token.arn,
+      SECRET_KEY_BASE       = data.aws_secretsmanager_secret.static_secret_key_base.arn,
+    },
+  )
+  log_group          = local.log_group
+  aws_region         = data.aws_region.current.name
+  cpu                = local.static_defaults.cpu
+  memory             = local.static_defaults.memory
+  task_role_arn      = aws_iam_role.task.arn
+  execution_role_arn = aws_iam_role.execution.arn
   load_balancers = [{
     target_group_arn = module.static_public_alb.target_group_arn
     container_port   = 80
   }]
-  environment_variables = {} # TODO
-  secrets_from_arns     = {} # TODO
-  log_group             = local.log_group
-  aws_region            = data.aws_region.current.name
-  cpu                   = 512
-  memory                = 1024
-  task_role_arn         = aws_iam_role.task.arn
-  execution_role_arn    = aws_iam_role.execution.arn
 }
 
 module "draft_static" {
+  source = "../../modules/app"
+
   service_name                     = "draft-static"
   mesh_name                        = aws_appmesh_mesh.govuk.id
   service_discovery_namespace_id   = aws_service_discovery_private_dns_namespace.govuk_publishing_platform.id
   service_discovery_namespace_name = aws_service_discovery_private_dns_namespace.govuk_publishing_platform.name
-  subnets                          = local.private_subnets
-  vpc_id                           = local.vpc_id
   cluster_id                       = aws_ecs_cluster.cluster.id
-  source                           = "../../modules/app"
+  vpc_id                           = local.vpc_id
+  subnets                          = local.private_subnets
   desired_count                    = var.draft_static_desired_count
   extra_security_groups            = [local.govuk_management_access_security_group, aws_security_group.mesh_ecs_service.id]
+  environment_variables = merge(
+    local.static_defaults.environment_variables,
+    {
+      ASSET_HOST          = "https://draft-static-ecs.${local.defaults.environment_variables.GOVUK_APP_DOMAIN_EXTERNAL}",
```

Comment:
As above - `${var.external_app_domain}` is a better reference for the external app domain.

Reply:
same as above.